base/applications/cmdutils/more/more.c

```diff
@@ -945,9 +949,17 @@ int wmain(int argc, WCHAR* argv[])
     }
 
     /* Load the registry settings */
-    LoadRegistrySettings(HKEY_LOCAL_MACHINE);
-    LoadRegistrySettings(HKEY_CURRENT_USER);
-    s_dwFlags = (s_bEnableExtensions ? FLAG_E : 0);
+    s_dwFlags = 0;
+    if (LoadRegistrySettings(HKEY_LOCAL_MACHINE, FALSE))
+    {
+        if (LoadRegistrySettings(HKEY_CURRENT_USER, TRUE))
+            s_dwFlags = FLAG_E;
+    }
+    else
+    {
+        if (LoadRegistrySettings(HKEY_CURRENT_USER, FALSE))
+            s_dwFlags = FLAG_E;
+    }
```

Comment:
The previous code was behaving correctly, so why did you change it?

Reply:
Reverted.